docker/services/simplified_crontab

```diff
@@ -97,10 +97,14 @@ HOME=/var/www/circulation
 0 0 2 * * root core/bin/run opds_for_distributors_reaper_monitor >> /var/log/cron.log 2>&1
 0 4 * * * root core/bin/run opds_for_distributors_import_monitor >> /var/log/cron.log 2>&1
 
-# Vanilla OPDS
+# Vanilla OPDS 1.x
 #
 0 5 * * * root core/bin/run opds_import_monitor >> /var/log/cron.log 2>&1
 
+# Vanilla OPDS 2.x
+#
+0 5 * * * root core/bin/run opds2_import_monitor >> /var/log/cron.log 2>&1
```

Comment:
Looks fine.

You might want to offset this a little to avoid kicking this off at the same time as `opds_import_monitor`. Not sure what the best amount of time would be, but I'd try 30 minutes, which would at least reduce some overlap.
```
30 5 * * * root core/bin/run opds2_import_monitor >> /var/log/cron.log 2>&1
```